readme.md

```diff
@@ -1,246 +0,0 @@
-# Grunt Email Design Workflow
-
-[![Built with Grunt](https://gruntjs.com/cdn/builtwith.png)](http://gruntjs.com/)
-
-Designing and testing emails is a pain. HTML tables, inline CSS, various devices and clients to test, and varying support for the latest web standards.
-
-This Grunt task helps simplify things.
-
-1. Compiles your SCSS to CSS
-
-1. Builds your HTML email templates
-
-1. Inlines your CSS
-
-1. Compresses and uploads images to a CDN such as Rackspace Cloud or AWS S3 (optional)
-
-1. Replaces the image URLs with the urls that point to the CDN images (optional)
-
-1. Sends a test email to your inbox or Litmus (optional)
-
-## Requirements
-
-You may already have these installed on your system. If not, you'll have to install them.
-
-* Node.js - [Install Node.js](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager)
-* NVM - [Install node version manager](https://github.com/nvm-sh/nvm)
-* Grunt-cli and Grunt (`npm install grunt-cli -g`)
-* [Mailgun](http://www.mailgun.com) (optional) - Sends the email
-* [Litmus](https://litmus.com) (optional) - Tests the email across all clients/browsers/devices
-* [Rackspace Cloud](http://www.rackspace.com/cloud/files/) (optional) - Uses Cloud Files as a CDN
-
-## Getting started
-
-If you haven't used [Grunt](http://gruntjs.com/) before check out Chris Coyier's post on [getting started with Grunt](http://24ways.org/2013/grunt-is-not-weird-and-hard/).
-
-#### 1. Setup
-
-Clone this repo, cd to the directory, run `nvm use` _(to set correct node version)_ and `npm install` to install the necessary packages.
-
-```sh
-git clone https://github.com/leemunroe/grunt-email-workflow.git
-cd grunt-email-workflow
-npm install
-```
-
-#### 2. Create secrets.json
-
-Create a `secrets.json` file in your project root as **outlined below under "[Sensitive Information](#sensitive-information)"**.
-
-#### 3. Run Grunt
-
-Run `grunt` in command line and check out your `/dist` folder to see your compiled and inlined email templates.
-
-### Sensitive information
```

Comment:
I think this info and example is important for people just getting started (open to rewording or making it clearer). Would recommend keeping Mailgun and S3 (per previous Grunt task comments).

Reply:
Done. Text updated.